Microsoft.Toolkit.Uwp/Helpers/BackgroundTaskHelper.cs

```diff
@@ -0,0 +1,107 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Background;
+
+namespace Microsoft.Toolkit.Uwp.Helpers
+{
+    /// <summary>
+    /// This class provides static helper methods for background task.
+    /// </summary>
+    public static class BackgroundTaskHelper
+    {
+        /// <summary>
+        /// Check if a background task has been registered with the application or not.
+        /// </summary>
+        /// <param name="backgroundTaskType">The type of the background task.</param>
+        /// <returns>True/False indicating if a background task was registered or not</returns>
+        public static bool IsBackgroundTaskRegistered(Type backgroundTaskType)
```

Comment:
Need an overload that simply accepts a name, so it can be used by Single Process Model, right? I know you currently use the type to create the name, but Single Process Model wouldn't have a type to provide, it just has a name.